### DISCUSSION TITLE
Plugins: How to alter migrations for moved models?

### DISCUSSION THREAD

**Discussion by alehaa** [netbox-community/netbox#17649] [Help Wanted!] [answered]
With https://github.com/netbox-community/netbox/issues/16388 the `ObjectChange` model moved from `extras` to `core`. In one of my migrations I've used the following code which apparently breaks when the migration is executed in a `v4.1` deployment.

```Python
ObjectChange = apps.get_model("extras", "ObjectChange")
```

To solve this I simply edited the migration and changed `extras` to `core`. However, this somehow feels like tweaking, as two different installations could see the migration with different code. Is this ok, as the state at the time of execution is relevant, or is there a better way to solve this retrospectively?

**Comment by jeremystretch** [comment, marked as answer]
I believe this is fine: `get_model()` will read the model as it's currently registered in the code base (i.e. in the "new" location) when migrations are run.
